src/SUMMARY.md

```diff
@@ -0,0 +1,16 @@
+# Summary
+
+[LazyLedger Specifications](./README.md)
+
+- [Specification](./specs/README.md)
+  - [Architecture](./specs/architecture.md)
+  - [Data Structures](./specs/data_structures.md)
+  - [Consensus](./specs/consensus.md)
+  - [Block Proposer](./specs/block_proposer.md)
+  - [Networking](./specs/networking.md)
+  - [Light client](./specs/light_client.md)
+  - [Node types](./specs/node_types.md)
+- [Rationale](./rationale/README.md)
```

Comment:
![image](https://user-images.githubusercontent.com/7441766/120081776-8f46cb00-c0bf-11eb-9ab9-7fe4202ea71a.png)

Reply:
Fixed in a3be997.